integration/model_aws_service.go

```diff
@@ -181,6 +181,6 @@ var AWSServiceNames = map[string]AwsService{
 	"AWS/WorkMail":                AWSWORK_MAIL,
 	"AWS/WorkSpaces":              AWSWORK_SPACES,
 	"AWS/Neptune":                 AWSNEPTUNE,
-	"AWS/MediaLive":               AWSMEDIA_LIVE,
+	"MediaLive":               AWSMEDIA_LIVE,
```

Comment:
Alignment is off.

Reply:
sorry, I did not checked the result of my sed command